internal/events/event.go

```diff
@@ -455,7 +471,7 @@ func (e Events) V2GetEvents(ctx context.Context, params *common.V2GetEventsParam
 		selectedCategories = append(selectedCategories, DefaultEventCategories...)
 	}
 
-	events, eventSeverityCount, err := e.queryEvents(ctx, selectedCategories, params.ClusterID, params.HostIds, params.InfraEnvID, params.Limit, params.Offset, params.Severities, params.Message, params.DeletedHosts, params.ClusterLevel)
+	events, eventSeverityCount, err := e.queryEvents(ctx, selectedCategories, params.ClusterID, params.HostIds, params.InfraEnvID, params.Limit, params.Offset, params.Order, params.Severities, params.Message, params.DeletedHosts, params.ClusterLevel)
```

Comment:
Not really relevant to this PR but I guess we should pass all params through instead having a long list